sound/soc/sof/intel/cnl.c

```diff
@@ -280,6 +280,14 @@ int cnl_ipc4_send_msg(struct snd_sof_dev *sdev, struct snd_sof_ipc_msg *msg)
 	snd_sof_dsp_write(sdev, HDA_DSP_BAR, CNL_DSP_REG_HIPCIDR,
 			  msg_data->primary | CNL_DSP_REG_HIPCIDR_BUSY);
 
+	if (SOF_IPC4_MSG_IS_MODULE_MSG(msg_data->primary) &&
```

Comment:
what are you checking here?

Reply:
check pm msg which is only defined in module. I will update comments